lib/her/model/associations.rb

```diff
@@ -26,12 +26,12 @@ def get_association(association_name)
       end
 
       module ClassMethods
-        # Return @_her_associations, lazily initialized with copy of the
-        # superclass' her_associations, or an empty hash.
+        # Return @associations, lazily initialized with copy of the
+        # superclass' associations, or an empty hash.
         #
         # @private
         def associations
-          @_her_associations ||= begin
+          @associations ||= begin
```

Comment:
I prefer this style as well but don't know what the effects would be or the original reason for prefixing these instance variables with `_her.  For example would we run into a situation [like @attributes](https://github.com/remiprev/her/pull/497) where another class defines `@associations` and we have a naming conflict? Any thoughts @edtjones?